services/blue/src/itops/Abstraction/NServiceBus/AdsdHotelMessageConventions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using artiso.AdsdHotel.ITOps.Communication;
+using NServiceBus;
+
+namespace artiso.AdsdHotel.ITOps.Abstraction.NServiceBus
+{
+    internal class AdsdHotelMessageConventions : IMessageConvention
```

Comment:
Reuse the class in the infrastructure project.

Reply:
With the consolidation of the projects, only one `AdsdHotelMessageConventions` will remain.